_config/debugbar.yml

```diff
@@ -51,3 +51,5 @@ SilverStripe\Admin\LeftAndMain:
 SilverStripe\Control\Controller:
   extensions:
     - LeKoala\DebugBar\Extension\ControllerExtension
+LeKoala\DebugBar\DebugBar:
+  show_namespaces: false
```

Comment:
This could just be another option in the definitions from line 9. Could you also name it something like "show namespaces in db profiling"?